.github/ISSUE_TEMPLATE/operate-feature.md

```diff
@@ -6,38 +6,33 @@ labels: ["kind/feature", "component/operate"]
 assignees: ''
 
 ---
-<!-- Remember to link this feature issue in the parent Epic -->
 
-### Description
-<!-- [Optional] A clear description of the story -->
+### Description (required on creation)
+
+<!-- A clear description of the feature -->
+
+### User story (Required on creation)
 
-### User Story
-<!-- [Mandatory field] -->
 ```Gherkin
 Scenario: As an Operate user
     Given I am logged in
     When I select one item/tab in the toolbar
     Then this item should be shown in highlighted (in bold)
 ```
+
 ### Designs
+
 <!-- [Optional] <Figma Link> -->
 
-### Acceptance Criteria
-<!-- [Mandatory field] The assignee will fill the Acceptance Criteria. -->
-- [ ]
+### Functional Requirements (required before implementation)
 
-### Definition of Ready - Checklist
-<!-- the assignee will check the DOR. -->
+<!-- Requirements that directly implement the user story -->
+ 
+### Technical Requirements (required before implementation)
 
-- [ ] The issue has a meaningful title, description, and testable acceptance criteria
-- [ ] If documentation needs to be updated, an issue is created in the [camunda-docs](https://github.com/camunda/camunda-docs) repo, and the issue is added to our Operate project board.
```

Comment:
I liked this reminder about documentation. For epics it makes more sense to have a docs section in the epic template, but for single features without epic this would be nice to have in my opinion.

Reply:
Hmm, I'd like to avoid having a process description in a ticket template. I'd rather have supporting documentation in e.g. Confluence then. E.g. what do I need to do to work with a feature request ticket: Do x, y, z.

With a process description in the template, it becomes noise as soon as you have internalized that knowledge and don't need the reminder anymore. And this should be the target and normal state.

Reply:
I have added a guide on how to write a good ticket to https://confluence.camunda.com/display/HAN/Operate+Development+Process